src/pages/javascript/with/index.md

```diff
@@ -3,13 +3,46 @@ title: With
 ---
 ## With
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/javascript/with/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+JavaScript's `with` statement is a shorthand way for editing several properties on one object. Most developers discourage usage of `with`, and you are best not using this keyword.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+**Note**: `"strict mode"` forbids usage of `with`.
+
+
+### Syntax
+```syntax
+with (expression)
+  statement
+```
+
+
+### Example Usage
+In JavaScript, you can individually modify an object's properties like below:
+```javascript
+earth.land.tree.kind = 'apple';
+earth.land.tree.height = 4;
```

Comment:
I like the example, but maybe create a reproducible one you can quickly run?

Something like

```js
var earth = {};
earth.gem = "Diamond";
earth.ocean = "Pacific";
```